### PR TITLE
Feature/pre healthkit ui updates

### PR DESCRIPTION
**Pull request by markyou**
These are some pre-req tasks before getting started on new React Native HealthKit sync UI for iOS.

Update most npm dependencies, fix storybook, fix some stories, fix storyshots, fix tests, update README, fix Android build, and fix an Android regression introduced in 3.0.1 (with the Expo 31 upgrade).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,65 +9,63 @@
...
     "format": "prettier-eslint \"src/**/*.js\"",
     "postinstall": "patch-package"
   },
   "dependencies": {
     "axios": "0.18.0",
-    "date-fns": "1.29.0",
-    "expo": "^31.0.2",
-    "expo-three": "^2.2.2-alpha.1",
+    "date-fns": "1.30.1",
+    "expo": "^31.0.4",
```
Comment (cbwebdevelopment) [inline package.json]: We should probably lock this version down.  Same with `react-native-walkthrough-tooltip` and `jest-expo`

File: src/components/Graph/gl/GraphBolusGl.js [hunk 2]
```diff
@@ -262,19 +267,24 @@ class GraphBolusGl extends GraphRenderLayerGl {
...
-      x: x + 0.5,
-      y: centerY,
-      z,
-      contentOffsetX: this.contentOffsetX,
-      pixelsPerSecond: this.pixelsPerSecond,
-      shouldScrollX: true,
-    });
+    const rectangleGeometryWidth = width - 1;
+    const rectangleGeometryHeight = height - 1;
+    console.log(`rendering bolus rect, width: ${width}, height: ${height}`);
```
Comment (cbwebdevelopment) [inline src/components/Graph/gl/GraphBolusGl.js]: Should this console log be commented out or removed?  I'm assuming any logs intentionally left would be called via `Logger`

Comment (markyou) [reply src/components/Graph/gl/GraphBolusGl.js]: Yes, sorry. Forgot to eslint it (which would have caught it). (Hmmm.. thought I had eslint enabled in Travis, will look.)

File: package.json (hunk 1 above)
Comment (markyou) [reply package.json]: Agreed, will do.

**Pull request merged**